### DISCUSSION TITLE
Clip volume lock (for presets)

### DISCUSSION THREAD

**Discussion by legsmechanical** [SynthstromAudible/DelugeFirmware#549] [Ideas]
Would like to see a setting that allows "clip volume" to override "preset volume" when loading/previewing synth/kit presets.  

WHAT: 

- When turned on, any preset you audition/load would automatically inherit the volume parameter value of the preset currently loaded in the clip.  As a result, the volume value stays constant across all presets unless/until it is manually changed.

 
WHY:

- Auditioning presets can be a pain because they are previewed/loaded with their volume set at whatever value they were saved with.  As a result you get wide variations in volume when scrolling through them (particularly problematic when you're trying to find a sound that fits in your existing mix).

**Comment by m-m-adams** [comment]
I don’t think this would  solve the problem - you’d still end up with volume swings from filter cutoff, oscillator shape and volume, fm depth, and various other parameters that effect the output volume. The saved patch volume is ideally used to balance out volume between presets

**Reply by legsmechanical** [reply]
Sure there are lots of other variables and this wouldn't give you a 1:1 across presets, but it’d absolutely get you closer to it for most presets.

**Comment by davidhaley** [comment]
+1 for this feature.

**Comment by juice721** [comment]
+1 !

When I've got a loop playing with a synth set at a volume set to match the mixing volume of the whole song, if I want to try other synth presets for this loop, the new ones jump lound in volume and this is very annoying ! This feature will fix this problem !
